Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -77,14 +84,18 @@ final class DefaultThemeManager: ThemeManager, Notifiable {
         guard currentTheme.type != newTheme else { return }
         // TODO: Check for nightmode FXIOS-4910
         currentTheme = newThemeForType(newTheme)
+        appDelegate?.window??.overrideUserInterfaceStyle = currentTheme.type.getInterfaceStyle()
 
         ensureMainThread { [weak self] in
             self?.notificationCenter.post(name: .ThemeDidChange)
         }
     }
 
     func systemThemeChanged() {
-        guard userDefaults.bool(forKey: ThemeKeys.systemThemeIsOn) else { return }
+        // Ignore if the system theme is off or night mode is on
+        guard userDefaults.bool(forKey: ThemeKeys.systemThemeIsOn),
+              let nightModeIsOn = userDefaults.object(forKey: ThemeKeys.NightMode.isOn) as? NSNumber,
+              nightModeIsOn.boolValue == false else { return }
```

Comment:
```suggestion
        // Ignore if the system theme is off or night mode is on
        guard userDefaults.bool(forKey: ThemeKeys.systemThemeIsOn),
              let nightModeIsOn = userDefaults.object(forKey: ThemeKeys.NightMode.isOn) as? NSNumber,
              nightModeIsOn.boolValue == false
        else { return }
```
nit: else blocks on new lines is the general style we've been going with for multi-line guard statements as it's a nice closer, visually.